packages/edge-config/src/index.common.test.ts

```diff
@@ -389,3 +389,32 @@ describe('connectionStrings', () => {
     });
   });
 });
+
+describe('in-memory cache with swr behaviour', () => {
+  const originalEnv = process.env.NODE_ENV;
+
+  beforeAll(() => {
+    process.env.NODE_ENV = 'development';
+  });
+
+  afterAll(() => {
+    process.env.NODE_ENV = originalEnv;
+  });
+
+  it('use in-memory cache', async () => {
+    fetchMock.mockResponse(JSON.stringify({ foo: 'bar' }));
+
+    const edgeConfig = pkg.createClient(
+      'https://edge-config.vercel.com/ecfg-2?token=token-2',
+    );
+    expect(await edgeConfig.get('foo')).toBe('bar');
+
+    fetchMock.mockResponse(JSON.stringify({ foo: 'bar2' }));
+    expect(await edgeConfig.get('foo')).toBe('bar'); // 1st call goes to the cache
+
+    await new Promise<void>((res) => {
+      setTimeout(res, 1000);
```

Comment:
Does this really need to be 1s? This would slow the tests down right?

Reply:
Not really, reduced it to 100ms.

It works even without any waiting, but less predictable, since we can't know when the promise gets resolved.